code/game/objects/items/storage/backpack.dm

```diff
@@ -110,6 +110,7 @@
 	desc = "It's useful for both carrying extra gear and proudly declaring your insanity."
 	icon_state = "backpack-cult"
 	inhand_icon_state = "backpack"
+	alternate_worn_layer = 4
```

Comment:
needs to use defines

Reply:
dumb question - what would that look like? i dont see anything similar in the backpack file. do you mean like putting in a define file #define OBVIOUS_BACKPACK 4 and then alternate_worn_layer = OBVIOUS_BACKPACK or am i way off here

Reply:
we have a file where all layers are defined. use one of those layers
i recommend you use the same thing modsuits do, which is `HANDS_LAYER+0.1` or `HEAD_LAYER-0.1`

Reply:
oh, didn't realize this was an option, will fix